### ISSUE TITLE
Request for new icon for the"Opening Remote Repository" Scenario

### ISSUE THREAD

**Issue by bhavyaus** [microsoft/vscode-codicons#149]
Hi, 

We have added a quick access icon on the primary side bar for Remote Repositories so that users can quickly open a remote github/azure repo. See [#139](https://github.com/microsoft/vscode-remote-repositories-github/issues/139) 

Currently, its the + icon which could confuse users since that implies "Adding/Creating" where as the action involved is "Opening"

![image](https://user-images.githubusercontent.com/25044782/201411274-a0dfa156-46de-4d78-8ee0-c02f737cd2c2.png)

Thank you!

**Comment by daviddossett** [comment]
Taking both `open-folder` and `vm-connect` icons as inspiration, here's what I propose (in welcome page context to show difference from `open-folder`. Thoughts?

<img width="499" alt="CleanShot 2022-11-21 at 16 02 39@2x" src="https://user-images.githubusercontent.com/25163139/203182397-aa754054-d661-412f-82c2-d74a713f2eb7.png">

Icons together for reference:

<img width="584" alt="CleanShot 2022-11-21 at 16 03 49@2x" src="https://user-images.githubusercontent.com/25163139/203182547-37118dbd-545a-4a8e-931d-b013729d157e.png">

**Comment by bhavyaus** [comment]
LGTM. It matches the remote explorer Activity Bar Icon. Any idea why the Remote Hub Extension Icon is 
<img width="231" alt="image" src="https://user-images.githubusercontent.com/25044782/203182869-5c74fa2e-3574-4b96-bf33-72e31f5591f7.png">

https://marketplace.visualstudio.com/items?itemName=ms-vscode.remote-repositories

I should probably change that up too.

**Comment by joyceerhl** [comment]
@daviddossett added that one in July this year https://github.com/microsoft/vscode-remote-repositories-github/issues/209#issuecomment-1184864802. I know we talked about doing a refresh of all the remote-ish extensions at some point though. The only concern I can think of is that the Open Remote Repository scenario does not take place over a remote connection, so that might be confusing to users (and is possibly why we went with the internet connection icon instead).

**Comment by daviddossett** [comment]
That's a fair point @joyceerhl. We can continue using that signal metaphor for this folder icon too. In fact, I don't know why I didn't think to use that icon here. Blame the Monday brain, I suppose.

It just needs to be modified for use at the "normal" codicon size.

**Comment by daviddossett** [comment]
A second pass:

<img width="666" alt="CleanShot 2022-11-21 at 16 55 37@2x" src="https://user-images.githubusercontent.com/25163139/203188394-1284eed5-5ff0-4c96-baa3-22e7523e1442.png">

<img width="647" alt="CleanShot 2022-11-21 at 16 56 20@2x" src="https://user-images.githubusercontent.com/25163139/203188383-1a1ae7b1-948e-4f84-a7a1-26a7676d829d.png">

**Comment by bhavyaus** [comment]
> @daviddossett added that one in July this year [microsoft/vscode-remote-repositories-github#209 (comment)](https://github.com/microsoft/vscode-remote-repositories-github/issues/209#issuecomment-1184864802). I know we talked about doing a refresh of all the remote-ish extensions at some point though. The only concern I can think of is that the Open Remote Repository scenario does not take place over a remote connection, so that might be confusing to users (and is possibly why we went with the internet connection icon instead).

But Remote Repositories is also accessible through the Remote Explorer which also is home to other remote development scenarios and that uses the >< symbols in the icon?

**Comment by joyceerhl** [comment]
Yeah that's a good point--the Remote Repos view has always been in the Remote Explorer, and its primary value add is to ensure that users do not lose their uncommitted changes in repositories that they have opened before. Maybe this means we reconsider whether 1. we need the view and 2. the view fits in the Remote Explorer, especially if we are storing uncommitted changes with Edit Sessions.

**Comment by mrleemurray** [comment]
Do we still require a specific icon for this?
